src/displayapp/screens/WatchFaceFuturistic.cpp

```diff
@@ -0,0 +1,182 @@
+  lv_arc_set_range(arcMin, 0, 60);
+  lv_obj_align(arcMin, lv_scr_act(), LV_ALIGN_CENTER, 0, 0);
+  lv_arc_set_rotation(arcMin, 270);
+  lv_arc_set_bg_angles(arcMin, 0, 360);
+  lv_arc_set_adjustable(arcMin, false);
+  lv_arc_set_start_angle(arcMin, 0);
+  lv_obj_set_style_local_line_color(arcMin, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, minutesArcColor);
+  lv_obj_set_style_local_line_width(arcMin, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, 8);
+  lv_obj_set_style_local_line_rounded(arcMin, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, false);
+  lv_obj_set_style_local_line_width(arcMin, LV_ARC_PART_BG, LV_STATE_DEFAULT, 0);
+
+  arcHour = lv_arc_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(arcHour, 140, 140);
+  lv_arc_set_range(arcHour, 0, 12);
+  lv_obj_align(arcHour, lv_scr_act(), LV_ALIGN_CENTER, 0, 0);
+  lv_arc_set_rotation(arcHour, 270);
+  lv_arc_set_bg_angles(arcHour, 0, 360);
+  lv_arc_set_adjustable(arcHour, false);
+  lv_arc_set_start_angle(arcHour, 0);
+  lv_obj_set_style_local_line_color(arcHour, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, hoursArcColor);
+  lv_obj_set_style_local_line_width(arcHour, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, 8);
+  lv_obj_set_style_local_line_rounded(arcHour, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, false);
+  lv_obj_set_style_local_line_width(arcHour, LV_ARC_PART_BG, LV_STATE_DEFAULT, 0);
+
+  timeDateLabel = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_color(timeDateLabel, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, innerTextColor);
+  lv_label_set_text_fmt(timeDateLabel, "[init]");
+  lv_label_set_align(timeDateLabel, LV_LABEL_ALIGN_CENTER);
+  lv_obj_align(timeDateLabel, nullptr, LV_ALIGN_CENTER, 0, -12);
+
+  taskRefresh = lv_task_create(RefreshTaskCallback, LV_DISP_DEF_REFR_PERIOD, LV_TASK_PRIO_MID, this);
+
+  Refresh();
+}
+
+WatchFaceFuturistic::~WatchFaceFuturistic() {
+  lv_task_del(taskRefresh);
+
+  lv_obj_clean(lv_scr_act());
+}
+
+void WatchFaceFuturistic::Refresh() {
+  heartbeat = heartRateController.HeartRate();
+  heartbeatRunning = heartRateController.State() != Controllers::HeartRateController::States::Stopped;
+  if (heartbeat.IsUpdated() || heartbeatRunning.IsUpdated()) {
+    if (heartbeatRunning.Get()) {
+      lv_obj_set_hidden(heartbeatIcon, false);
+      lv_obj_set_hidden(heartbeatValue, false);
+      lv_label_set_text_fmt(heartbeatValue, "%03d", heartbeat.Get());
+    } else {
+      lv_obj_set_hidden(heartbeatIcon, true);
+      lv_obj_set_hidden(heartbeatValue, true);
+    }
+
+    lv_obj_realign(heartbeatIcon);
+    lv_obj_realign(heartbeatValue);
+  }
+
+  batteryPercentRemaining = batteryController.PercentRemaining();
+  isCharging = batteryController.IsCharging();
+  if (isCharging.IsUpdated() || batteryPercentRemaining.IsUpdated()) {
+    if (isCharging.Get()) {
+      lv_obj_set_hidden(plugIcon, false);
+      lv_obj_set_hidden(batteryStatus, false);
+      lv_label_set_text_fmt(batteryStatus, "%03d%%", batteryPercentRemaining.Get());
```

Comment:
I wouldn't force leading zeroes here and probably not in the bpm either. It looks a bit weird to me.